### PR TITLE
Improve readme migration namespace

### PR DESCRIPTION
**Pull request by bscheshirwork**

### CODE REVIEW

**Comment by bscheshirwork** [comment]
related #66

File: changelog.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # Gallery manaager extension changelog
 
+Nov 19, 2018
+  - Use migrations config from `console/config/main.php` instead of create migration directly 
+
```
Comment (zxbodya) [inline changelog.md]: I think, changelog should more about important changes in package itself… and readme update does not look to belong here

Comment (bscheshirwork) [reply changelog.md]: Of course.
Change it as you wish.

Comment (bscheshirwork) [reply changelog.md]: The readme.md will be improve. Will add different way for apply migrations. This changes should be reflected for existing users who already use old way.

**Pull request closed without merging**